drupal-org.make

```diff
@@ -96,3 +97,7 @@ libraries[colorbutton][type] = library
 libraries[colorbutton][download][type] = get
 libraries[colorbutton][download][url] = http://download.ckeditor.com/colorbutton/releases/colorbutton_4.7.0.zip
 libraries[colorbutton][destination] = libraries
+libraries[btbutton][type] = library
+libraries[btbutton][download][type] = get
+libraries[btbutton][download][url] = https://github.com/smonetti/btbutton/archive/master.zip
```

Comment:
Please, add hashed version of library when it is dev. We can't rely on every dev in the future. It should be specified one